### PR TITLE
multitail: 7.1.3 -> 7.1.5

### PR DESCRIPTION
**Pull request by matthiasbeyer**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/tools/misc/multitail/default.nix [hunk 1]
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp bin/multitail $out/bin
+    cp multitail $out/bin/
```
Comment (Aleksanaa) [inline pkgs/tools/misc/multitail/default.nix:20]:
```suggestion
    install -Dm755 multitail -t $out/bin/
```

File: pkgs/tools/misc/multitail/default.nix [hunk 2]
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
```
Comment (Aleksanaa) [inline pkgs/tools/misc/multitail/default.nix:19]:
```suggestion
```

File: pkgs/tools/misc/multitail/default.nix [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-  version = "7.1.3";
+  version = "7.1.5";
   pname = "multitail";
 
   src = fetchFromGitHub {
     owner = "folkertvanheusden";
     repo = pname;
     rev = version;
-    sha256 = "sha256-4iAFxDAS3gthYPECzyFj0AKzIGwZ9hTJ96fCnQGLbXU=";
+    sha256 = "sha256-c9NlQLgHngNBbADZ6/legWFaKHJAQR/LZIfh8bJoc4Y=";
```
Comment (Aleksanaa) [inline pkgs/tools/misc/multitail/default.nix:11]:
```suggestion
    hash = "sha256-c9NlQLgHngNBbADZ6/legWFaKHJAQR/LZIfh8bJoc4Y=";
```

File: pkgs/tools/misc/multitail/default.nix [hunk 4]
```diff
@@ -1,23 +1,22 @@
...
   };
 
   nativeBuildInputs = [ pkg-config cmake ];
 
   buildInputs = [ ncurses ];
 
   installPhase = ''
-    mkdir -p $out/bin
-    cp bin/multitail $out/bin
+    install -Dm755 multitail -t $out/bin/
```
[the remark this answers is not in the document]
Comment (matthiasbeyer) [reply pkgs/tools/misc/multitail/default.nix:21]: Don't worry, my git kung fu is good enough to squash away all the little happenings :laughing:

**Pull request merged**